davitpy/pydarn/radar/tdiff/test_tdiff.py

```diff
@@ -182,7 +182,7 @@ def test_calc_tdiff(file_type="fitacf", password=True, tdiff_plot=None):
     In[5]: tt, trange, ldist = dtdiff.test_tdiff.test_calc_tdiff()
     --- shows simplex output ---
     In[6]: print tt
```

Comment:
Changing elevation angle and latitude calculation caused a slightly different tdiff value.